doc.go

```diff
@@ -0,0 +1,71 @@
+/*
+SQLair is a convenience layer for SQL databases that embeds Go structs directly into SQL queries.
+
+The SQL syntax is expanded with SQLair input and output expressions which indicate parts of the query that correspond to Go structs.
+This allows the user to specify the Go structs they want in the SQL query itself whilst allowing the full power of SQL to be utilised.
+This package also provides an alternative API for reading the rows from the database.
+It does not copy database/sql, it instead improves upon it and removes inconsistencies.
```

Comment:
Maybe: `It relies on database/sql for all the underneath operations.` or something similar.

This makes it clear that not only we're not copying it, we're relying on it.

Reply:
Changed.